src/framework/auth/strategies/dummy/dummy-strategy-options.ts

```diff
@@ -7,7 +7,6 @@ import { NbAuthStrategyOptions } from '../auth-strategy-options';
 import { NbAuthSimpleToken } from '../../services/';
 
 export class NbDummyAuthStrategyOptions extends NbAuthStrategyOptions {
-  name: string;
```

Comment:
why do we need to remove this name?

Reply:
@nnixaa 
As for the moment `requireValidToken` was set at strategy level, i put it in the ancestor `NbAuthStrategyOption`.
I then realized that only `NbPasswordStrategyOption` was extending `NbAuthStrategyOption`.
I guess it would be better (even if `requireValidToken` finally goes at endpoint level) to make OAuth2 and Dummy options also extend `NbAuthStrategyOption`.

In this case `name` (and probably more attributes) can be declared in `NbAuthStrategyOption`.

What do you think ?

Reply:
@alain-charles I'm bit confused, as far as I can see, all strategies' options (oauth2, dummy, password) are extended from `NbAuthStrategyOptions`. Am I missing something?

Reply:
@nnixaa  Have a look at `NbOAuth2AuthStrategyOptions` in the master branch